CHANGELOG.md

```diff
@@ -15,6 +15,98 @@ be exported in other repos. The issue is tracked
 [here](https://github.com/kubernetes/kubernetes/issues/48209#issuecomment-314537745).
 During the transition, you'll have to copy the helper functions to your projects. 
 
+**New features:**
+
+* Add paging support
+
+   * [https://github.com/kubernetes/kubernetes/pull/51876](https://github.com/kubernetes/kubernetes/pull/51876)
+
+* Add support for client-side spam filtering of events
+
+   * [https://github.com/kubernetes/kubernetes/pull/47367](https://github.com/kubernetes/kubernetes/pull/47367)
+
+* Support http etag and caching
+
+   * [https://github.com/kubernetes/kubernetes/pull/50404](https://github.com/kubernetes/kubernetes/pull/50404)
+
+* Extend SetHeader Requests method ito accept multiple values
+
+   * [https://github.com/kubernetes/kubernetes/pull/50639](https://github.com/kubernetes/kubernetes/pull/50639)
+
+* Add a heap to client-go
```

Comment:
Could you change it to "Adding priority queue support to informer cache."? That's the main use case.